project/src/components/offer-list/offer-list.tsx

```diff
@@ -0,0 +1,26 @@
+import {useState} from 'react';
+import {Offers} from '../../types/offer';
+import Card from '../card/card';
+
+type OfferListProps = {
+  offers: Offers;
+  className: string;
+}
+
+export default function OfferList({offers, className}: OfferListProps): JSX.Element {
+
+  const [activeCard, setActiveCard] = useState<number | null>(null);
```

Comment:
Нам пока активный айди не нужен, можно не извлекать
```suggestion
  const [, setActiveCard] = useState<number | null>(null);
```